src/main/java/org/folio/circulation/domain/notice/NoticeContextUtil.java

```diff
@@ -79,6 +80,8 @@ private static JsonObject createItemContext(Item item) {
     String captionsToken = JsonStringArrayHelper.toStream(item.getCopyNumbers())
       .collect(Collectors.joining("; "));
 
+    Optional<Location> location = Optional.ofNullable(item.getLocation());
```

Comment:
I'm a little surprised this wasn't implemented with an `if` statement rather than using optional. Can you help me understand why?

Reply:
I decided to use optional here because of conciseness, the number of lines remained the same. With an if statement the variable must be introduced. On the other hand I understand that using an if statement more effective. Changing the implementation with an if statement.